ERROR - 2025-08-26 08:49:00 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 08:52:32 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 08:56:23 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(39): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:09:17 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateCoursesTable.php(19): CodeIgniter\Database\Forge->createTable('courses')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-26 09:17:50 --> mysqli_sql_exception: Can't create table `lms_signar`.`enrollments` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateEnrollmentsTable.php(21): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-26 09:18:53 --> mysqli_sql_exception: Can't create table `lms_signar`.`enrollments` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateEnrollmentsTable.php(21): CodeIgniter\Database\Forge->createTable('enrollments')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRefresh.php(87): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-26 09:21:34 --> mysqli_sql_exception: Can't create table `lms_signar`.`enrollments` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateEnrollmentsTable.php(21): CodeIgniter\Database\Forge->createTable('enrollments', true, Array)
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:22:26 --> mysqli_sql_exception: Can't create table `lms_signar`.`enrollments` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateEnrollmentsTable.php(23): CodeIgniter\Database\Forge->createTable('enrollments', true, Array)
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:28:02 --> mysqli_sql_exception: Can't create table `lms_signar`.`enrollments` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateEnrollmentsTable.php(20): CodeIgniter\Database\Forge->createTable('enrollments', true)
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:29:40 --> mysqli_sql_exception: Can't create table `lms_signar`.`enrollments` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateEnrollmentsTable.php(23): CodeIgniter\Database\Forge->createTable('enrollments', true, Array)
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateEnrollmentsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:33:45 --> mysqli_sql_exception: Can't create table `lms_signar`.`courses` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `c...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `c...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `c...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `c...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-083237_CreateCoursesTable.php(21): CodeIgniter\Database\Forge->createTable('courses', true)
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateCoursesTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:39:05 --> mysqli_sql_exception: Invalid default value for 'created_at' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `u...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `u...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-18-144311_CreateAllTables.php(21): CodeIgniter\Database\Forge->createTable('users', true)
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateAllTables->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:46:45 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(45): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:49:54 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(45): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 09:52:09 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(45): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 10:12:44 --> mysqli_sql_exception: Unknown table 'lms_signar.submissions' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `sub...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `sub...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `sub...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `sub...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-094423_CreateSubmissionsTable.php(27): CodeIgniter\Database\Forge->dropTable('submissions')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 10:13:55 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(45): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 11:56:38 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(38): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:02:46 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:03:02 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:03:13 --> mysqli_sql_exception: Table 'lms_signar.users' doesn't exist in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\UserSeeder.php(33): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:06:48 --> mysqli_sql_exception: Unknown table 'lms_signar.submissions' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `sub...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `sub...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `sub...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `sub...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-094423_CreateSubmissionsTable.php(27): CodeIgniter\Database\Forge->dropTable('submissions')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-26 12:09:30 --> mysqli_sql_exception: Unknown table 'lms_signar.submissions' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `sub...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `sub...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `sub...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `sub...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-094423_CreateSubmissionsTable.php(27): CodeIgniter\Database\Forge->dropTable('submissions')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-26 12:13:18 --> mysqli_sql_exception: Unknown table 'lms_signar.submissions' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `sub...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `sub...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `sub...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `sub...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-094423_CreateSubmissionsTable.php(24): CodeIgniter\Database\Forge->dropTable('submissions')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\BaseCommand.php(119): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRefresh.php(86): CodeIgniter\CLI\BaseCommand->call('migrate:rollbac...', Array)
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#14 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#15 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#16 {main}
ERROR - 2025-08-26 12:27:59 --> mysqli_sql_exception: Can't create table `lms_signar`.`submissions` (errno: 150 "Foreign key constraint is incorrectly formed") in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('CREATE TABLE `s...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `s...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `s...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(570): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `s...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-122212_CreateSubmissionsTable.php(22): CodeIgniter\Database\Forge->createTable('submissions')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateSubmissionsTable->up()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(182): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:35:39 --> mysqli_sql_exception: Unknown table 'lms_signar.quizzes' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `qui...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `qui...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `qui...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `qui...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-122206_CreateQuizzesTable.php(24): CodeIgniter\Database\Forge->dropTable('quizzes')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:35:59 --> mysqli_sql_exception: Unknown table 'lms_signar.quizzes' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `qui...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `qui...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `qui...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `qui...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-122206_CreateQuizzesTable.php(24): CodeIgniter\Database\Forge->dropTable('quizzes')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:36:54 --> mysqli_sql_exception: Unknown table 'lms_signar.quizzes' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `qui...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `qui...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `qui...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `qui...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-122206_CreateQuizzesTable.php(24): CodeIgniter\Database\Forge->dropTable('quizzes')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:37:35 --> mysqli_sql_exception: Unknown table 'lms_signar.quizzes' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `qui...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `qui...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `qui...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `qui...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-122206_CreateQuizzesTable.php(24): CodeIgniter\Database\Forge->dropTable('quizzes')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 12:39:23 --> mysqli_sql_exception: Unknown table 'lms_signar.quizzes' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('DROP TABLE `qui...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('DROP TABLE `qui...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('DROP TABLE `qui...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Forge.php(663): CodeIgniter\Database\BaseConnection->query('DROP TABLE `qui...')
#4 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Migrations\2025-08-26-122206_CreateQuizzesTable.php(24): CodeIgniter\Database\Forge->dropTable('quizzes')
#5 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(876): App\Database\Migrations\CreateQuizzesTable->down()
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MigrationRunner.php(289): CodeIgniter\Database\MigrationRunner->migrate('down', Object(stdClass))
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\MigrateRollback.php(101): CodeIgniter\Database\MigrationRunner->regress(0)
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\MigrateRollback->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('migrate:rollbac...', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:00:02 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`courses`, CONSTRAINT `courses_instructor_id_foreign` FOREIGN KEY (`instructor_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\CoursesSeeder.php(23): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\CoursesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:00:25 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`courses`, CONSTRAINT `courses_instructor_id_foreign` FOREIGN KEY (`instructor_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\CoursesSeeder.php(23): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\CoursesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:04:39 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`enrollments`, CONSTRAINT `enrollments_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `en...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `en...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `en...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `en...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\EnrollmentsSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\EnrollmentsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:04:56 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`enrollments`, CONSTRAINT `enrollments_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `en...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `en...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `en...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `en...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\EnrollmentsSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\EnrollmentsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:06:53 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`enrollments`, CONSTRAINT `enrollments_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `en...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `en...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `en...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `en...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\EnrollmentsSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\EnrollmentsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:08:07 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`enrollments`, CONSTRAINT `enrollments_course_id_foreign` FOREIGN KEY (`course_id`) REFERENCES `courses` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `en...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `en...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `en...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `en...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\EnrollmentsSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\EnrollmentsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:11:32 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`enrollments`, CONSTRAINT `enrollments_course_id_foreign` FOREIGN KEY (`course_id`) REFERENCES `courses` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `en...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `en...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `en...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `en...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\EnrollmentsSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\EnrollmentsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:12:17 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`lessons`, CONSTRAINT `lessons_course_id_foreign` FOREIGN KEY (`course_id`) REFERENCES `courses` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `le...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `le...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `le...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `le...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\LessonsSeeder.php(23): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\LessonsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:12:42 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`lessons`, CONSTRAINT `lessons_course_id_foreign` FOREIGN KEY (`course_id`) REFERENCES `courses` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `le...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `le...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `le...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `le...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\LessonsSeeder.php(23): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\LessonsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:13:15 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(25): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:15:06 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(25): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:18:02 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(25): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:21:35 --> mysqli_sql_exception: Unknown column 'description' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:22:29 --> mysqli_sql_exception: Unknown column 'description' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:23:06 --> mysqli_sql_exception: Unknown column 'description' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(21): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:24:26 --> mysqli_sql_exception: Unknown column 'title' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `qu...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `qu...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `qu...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `qu...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\QuizzesSeeder.php(19): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\QuizzesSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:32:35 --> mysqli_sql_exception: Unknown column 'score' in 'field list' in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `su...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `su...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `su...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `su...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\SubmissionsSeeder.php(23): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\SubmissionsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-08-26 13:35:33 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`lms_signar`.`submissions`, CONSTRAINT `submissions_user_id_foreign` FOREIGN KEY (`user_id`) REFERENCES `users` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `su...', 0)
#1 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `su...')
#2 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `su...')
#3 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `su...', NULL, false)
#4 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-SIGNAR\app\Database\Seeds\SubmissionsSeeder.php(27): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-SIGNAR\system\Database\Seeder.php(149): App\Database\Seeds\SubmissionsSeeder->run()
#7 C:\xampp\htdocs\ITE311-SIGNAR\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-SIGNAR\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-SIGNAR\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-SIGNAR\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
